File: job-management/Domain/SubmittedJobConfiguration/JobConfig.cs
using System;
using System.Collections.Generic;

namespace Domain.SubmittedJobConfiguration
{
    public class JobConfig
    {
        public Guid JobId { get; set; }
        public List<string> DataAnalysers { get; set; }
        public List<string> DataAcquirers { get; set; }
        public string TopicQuery { get; set; }

        public JobStatus JobStatus { get; set; }
    }
}

File: storage/internal-storage/src/main/java/cz/cuni/mff/socneto/storage/internal/data/model/JobConfig.java
package cz.cuni.mff.socneto.storage.internal.data.model;

import lombok.Data;

import javax.persistence.*;
import java.util.List;
import java.util.UUID;

@Data
@Entity
public class JobConfig {
    @Id
    private UUID jobId;
    @ElementCollection
    @CollectionTable(name = "data_analysers", joinColumns = @JoinColumn(name = "jobId"))
    @Column(name = "dataAnalysers")
    private List<String> dataAnalysers;
    @ElementCollection
    @CollectionTable(name = "data_acquirers", joinColumns = @JoinColumn(name = "jobId"))
    @Column(name = "dataAcquirers")
    private List<String> dataAcquirers;
    private String topicQuery;
    private String status;
}
